sources/code/main/modules/extensions.ts

```diff
@@ -199,4 +201,4 @@ export async function loadChromiumExtensions(session:Electron.Session) {
 export const styles = Object.freeze({
   load: loadStyles,
   add: addStyle
-});
\ No newline at end of file
+});
```

Comment:
I think you messed with the newlines there. Please avoid doing that.

Reply:
Should be fixed now, my editor does this be default. I'm curious though why this project chooses not to use trailing newlines.

Reply:
I guess this was a bit opinionated choice, I had no need for a trailing newlines if they don't change anything for me and with that I fell they were for me like adding additional space at the end of the each line (and you can also save 1 byte on removing it 😆). On the other hand, thanks to some toolkit constantly messing with it and very little toolkit that removes the newlines at the end of the files, I think it might not be so consistent.

Again, I don't pull PRs that mess around it (since changes like this are off-topic), unless PR is meant to introduce some conventions around formatting or to fix it in the files that don't match them.